fastparse/shared/src/main/scala/fastparse/parsers/Terminals.scala

```diff
@@ -23,6 +23,10 @@ object Terminals {
     override val toString = "Pass"
   }
 
+  case class PassWith[T, ElemType, R](t: T) extends Parser[T, ElemType, R]{
```

Comment:
This should have a scaladoc comment, even if it's a short one-liner. Psychologically it makes a big difference to someone browsing the code or API in their editor, as it's a signal that this is something that you should use. 

In Scala, unlike in Java, libraries tend to be less strict about enforcing the private/public distinction, so the existence of documentation takes the role of indicating whether a function or class is something you should use, or whether it's some internal thing you can touch but shouldn't